x/stakeibc/keeper/unbonding_records_get_host_zone_unbondings_msgs_test.go

```diff
@@ -351,6 +365,42 @@ func (s *KeeperTestSuite) TestUnbondFromHostZone_Successful_UnbondTotalGreaterTh
 	s.CheckUnbondingMessages(tc, expectedUnbondings)
 }
 
+func (s *KeeperTestSuite) TestUnbondFromHostZone_Successful_RefreshedNativeAmount() {
```

Comment:
Should we do more unit testing? Feels like this adds a bit of coverage but doesn't cover everything

Reply:
100%, this is just adding coverage to the existing code, but we should definitely test the migrations very thoroughly